ui/src/components/ActuatorItem.vue

```diff
@@ -0,0 +1,77 @@
+<script setup lang="ts">
+const props = defineProps({
+  name: {
+    type: String,
+    required: true,
+  },
+  data: {
+    type: Object,
+    required: true,
+  },
+});
+
+/**
+ * Convert given bytes to 2 digits precision round exponent version string.
+ * @param bytes number
+ */
+function convertBytes(bytes: number): string {
```

Comment:
this seems like a helper function to me, please move to helper/utils and add test

Reply:
It's a one of AFAIK so local helper